stock_reserve/models/stock_reserve.py

```diff
@@ -107,8 +107,11 @@ def get_location_from_ref(self, ref):
 
     @api.model
     def _default_picking_type_id(self):
-        ref = "stock.picking_type_out"
-        return self.env.ref(ref, raise_if_not_found=False).id
+        return (
+            self.env["stock.picking.type"]
+            .search([("code", "=", "outgoing")], limit=1)
```

Comment:
Limit to the reserve company, as there can be several allowed companies when doing this.